enterprise/plugins/com.liferay.ide.studio.ui/src/com/liferay/ide/studio/ui/StudioPlugin.java

```diff
@@ -51,7 +51,7 @@
     // The plugin ID
     public static final String PLUGIN_ID = "com.liferay.ide.studio.ui"; //$NON-NLS-1$
 
-    public static final String PRODUCT_ID = PLUGIN_ID + ".product";
+    public static final String PRODUCT_ID = PLUGIN_ID + ".studio";
```

Comment:
This should still be .product.  we may have to change it in other places to match